src/runner.ts

```diff
@@ -218,6 +218,14 @@ async function doLinting(
 
     let lastFolder: string | undefined;
     let configFile: IConfigurationFile | undefined;
+    const isFileExcluded = (filepath: string) => {
+        if (configFile === undefined || configFile.linterOptions == null || configFile.linterOptions.exclude == null) {
+            return false;
+        }
+        const fullPath = path.resolve(filepath);
+        return configFile.linterOptions.exclude.some((pattern) => new Minimatch(pattern).match(fullPath));
```

Comment:
The excluded paths should probably be resolved relative to the location of `tslint.json`.
If `/foo/tslint.json` excludes `baz.ts` we should only exclude `/foo/baz.ts` and still lint `/foo/bar/baz.ts`.

That will only work when `filepath` is an absolute path.